dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/cdi/FutureEMBuilder.java

```diff
@@ -272,7 +287,7 @@ public EntityManagerBuilder createEMBuilder() {
         if (namespace == Namespace.COMP && metadataIdentifier.startsWith("EJB#"))
             throw exc(IllegalArgumentException.class,
                       "CWWKD1061.comp.name.in.ejb",
-                      repositoryInterface.getName(),
+                      repositoryInterfaces,
```

Comment:
Previously the user would see this:
`The org.example.mycompany.data.repositories.Products repository interface which is defined in the MyEJB enterprise bean module of the MyApp application configures a java:comp value for the dataStore attribute.`

Now they will see:
`The [class org.example.mycompany.data.repositories.Products] repository interfaces which are defined in the MyEJB enterprise bean module of the MyApp application configures a java:comp value for the dataStore attribute.`

At a minimum we should get rid of the extra "class" wording in the message, which would be a trivial change.
After that, there are a couple of directions we can choose from:
1. Make this and the other similar messages plural.
2. Duplicate this and the other similar messages, so that for each there is a single and a plural message to choose from.
3. Keep the singular messages only, and log an error message per repository interface name.
4. Keep the singular messages only, and log one error message for the first repository interface name only.

For this PR, go with 3 or 4 so that it doesn't require an NLS review until after we decide what we really want:
```suggestion
                      repositoryInterfaces.get(0).getName(), // TODO revisit this and other NLS messages
```